### PR TITLE
changed class and id names from camelCase to snake-case in html and c…

### PR DESCRIPTION
**Pull request by dingogap**
I updated index.html and style.css, changing Class Names and Ids using camelCase to snake-case to improve code readability and adherence to standards.

### CODE REVIEW

**Review by tkdgns0630 — APPROVED** [review]
LGTM

**Review by mscamposmg — COMMENTED** [review]
LGTM

File: index.html [hunk 1]
```diff
@@ -22,7 +22,7 @@ <h3>Movie Search</h3>
 
         <!-- Div for Search Field for Movie Name -->
         <div class="row">
-            <input id="findMovie" placeholder="Type Movie Name" type="text" class="validate">
+            <input id="find-novie" placeholder="Type Movie Name" type="text" class="validate">
```
Comment (Devarasen) [inline index.html:25]: spelling error maybe?

File: assets/css/style.css [hunk 2]
```diff
@@ -1,12 +1,12 @@
-.mlBtn {
```
Comment (Devarasen) [inline assets/css/style.css:1]: JS - Camel case - camelCase
HTML & CSS - Snake case - snake_case

thats the naming convention we're supposed to use

Dan mentions this in the API 2 lecture around 2:08:00

**Pull request merged**